neue-juristische-wochenschrift.csl

```diff
@@ -109,7 +109,7 @@
             <text macro="firstpage-locator"/>
           </group>
         </if>
-        <else-if type="article-newspaper">
+        <else-if type="article-newspaper post-weblog webpage song">
```

Comment:
Thank you. Funnily, it worked fine without the match="any". I added it
anyway, of course.